src/main/java/com/scaleset/cfbuilder/beanstalk/Application.java

```diff
@@ -0,0 +1,22 @@
+package com.scaleset.cfbuilder.beanstalk;
```

Comment:
Shouldn't this package be called `elasticbeanstalk` ? As in `AWS::ElasticBeanstalk::Application`